streaming/java/streaming-runtime/pom.xml

```diff
@@ -69,6 +69,21 @@
   <artifactId>protobuf-java</artifactId>
   <version>3.8.0</version>
 </dependency>
+<dependency>
```

Comment:
Why add typesafe? We didn't use it.

Reply:
We use RayConfig in LoggingUtil, RayConfig depends on typesafe. Bazel doesn't allow indirect dependency.